src/canon/Chemistry.jl

```diff
@@ -6,8 +6,8 @@ using ..Canon
 using Markdown
 
 @docapode("Brusselator"
-  ,"https://en.wikipedia.org/wiki/Brusselator"
-  ,"A model of reaction-diffusion for an oscillatory chemical system."
+  ,""
+  ,"[prigogine_symmetry_1968](@cite)"
```

Comment:
Having the text that was deleted and the wikipedia link would be good when available.